sdk/Lusid.Sdk.Tests/tutorials/Instruments/ComplexInstruments.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Misc
```

Comment:
Can you update the namespace to Lusid.Sdk.Tests.Tutorials.Instruments

Reply:
yes, done.